pkg/invoice/response.go

```diff
@@ -0,0 +1,31 @@
+package invoice
+
+import "time"
+
+// Response is the response from the Invoice API.
+type Response struct {
+	CurrencyID        string  `json:"currency_id"`
+	ExternalReference string  `json:"external_reference"`
+	PaymentMethodID   string  `json:"payment_method_id"`
+	PreApprovalID     string  `json:"preapproval_id"`
```

Comment:
se o json tag é `preapproval_id`, o campo deve ser assim
```suggestion
	PreapprovalID     string  `json:"preapproval_id"`
```